nexus/types/src/quiesce.rs

```diff
@@ -487,12 +617,12 @@ impl SagaQuiesceHandle {
     }
 }
 
-impl SagaQuiesceInner {
+impl SagaQuiesceStatus {
     /// Returns whether sagas are fully drained
     ///
     /// This condition is not permanent.  New sagas can be re-assigned to this
     /// Nexus.
-    pub fn is_fully_drained(&self) -> bool {
+    fn is_fully_drained(&self) -> bool {
```

Comment:
Is it possible this returns true if we have never tried to reassign sagas?

Reply:
@smklein That's correct (your statement is correct, and I believe it's also the correct behavior).  Think of "is_fully_drained()" as "there are no sagas running and none will ever run again _unless_ there's a subsequent re-assignment".  (It is _not_: "there are no sagas running and none will ever run again".  This is what we spent so much time in RFD 588 _trying_ to achieve but found it at odds with the idea that "but you might need to expunge a Nexus at any point".  If you want this condition, you really want `fully_drained_blueprint()`.)